src/sql/workbench/parts/query/common/resultsGridContribution.ts

```diff
@@ -16,7 +16,7 @@ const configurationRegistry = <IConfigurationRegistry>Registry.as(Extensions.Con
 const resultsGridConfiguration: IConfigurationNode = {
 	id: 'resultsGrid',
```

Comment:
nit: Consider updating this too - resultsGridAndMessages

Reply:
Oh just noticed your other commend about breaking backwards compat. I assume updating the name means that any users who had changed the value would lose their settings? If so not important enough to warrant the break IMO

Reply:
Yeah if they had changed the settings, changing the key would invalidate their change. Probably not a huge deal if we just call it out in release notes, but since this isn't a wide spread ask figure I would avoid it.